### ISSUE TITLE
Adding "tip" AFTER question has been given?

### ISSUE THREAD

**Issue by KlavsKlavsen** [inducer/relate#1221]
I am using relate to build educational series.. so point is that actually learn the correct answer if they respond incorrectly.. so I'd like to show an extra text .. after they've answered each question.. ( no matter if they get it right or wrong).

I can't seem to find any support for this? (a completion_text - per question kinda :)

**Comment by inducer** [comment]
You're right, this is not something that's supported right now. It lines up somewhat well with my plans, in that I would like to make it possible to allow grading consideration of multiple attempts at a single question within a single flow session. (Along the lines of "get full credit if you get it right on the first try, get 60% credit on the second, ...".) Displaying a hint alongside this behavior would be a natural fit.

That said, all that is mostly vaporware right now. One part of the plan that does exist is https://github.com/inducer/starlark-pyo3/, which is the direction in which I would like to take grading policies.

If you would like to help make it happen, I would be more than happy to discuss.

**Comment by KlavsKlavsen** [comment]
unfortunately its not on the shortlist.. I've put a small team on getting relate up and running in our company - to start teaching employees.. and we'll then slowly get courses created.. and there the need for "retry if you did it wrong" - is something we would appreciate too.. (as our goal is actually not to grade as much IF they learned it - but to help them learn it).. Security policies, Quality management in the company and so on. .Important stuff that everyone needs to know :)

So in 2025 - this need should come closer and hopefully the team finds the time and need for this - and then we'll pick up here - for a talk about how it could be done - as we'll gladly implement and submit a PR if we can find a way to do it
